Paganism/PParser/Parser.cs

```diff
@@ -28,6 +30,7 @@ public Parser(Token[] tokens, string filePath)
         public bool InLoop { get; private set; }
 
         public string Filepath { get; private set; }
+        public string _ExtensionFunction { get; private set; } = string.Empty;
```

Comment:
What problem with field name

Reply:
?

Reply:
to separate the name from the other common names and yeah should be internal

Reply:
Fields can start with '_' only if it is private

Reply:
in other ways using '_' it is wrong

Reply:
So also return public

Reply:
should not be public becauseof it's nature and use, let's remove the _ but keep it internal